### PR TITLE
Add first subdomain blacklist

### PR DESCRIPTION
**Pull request by meshy**

### CODE REVIEW

**Comment by meshy** [comment]
@wildfoundry/web-developers please review.

If you can think of a meaningful test to write for this, I'll happily add a test suite.

File: naughty/subdomains.py [hunk 1]
```diff
@@ -0,0 +1,341 @@
...
+    Feel free to fork/copy/whatever you want to do with this list. I release
+    any data in this repository into the public domain, with the understanding
+    that I take no responsibility for what is done with it, I provide no
+    warranty or guarantee for anything, etc.
+
+Many thanks to @ckrailo for making this module possible.
+"""
+
+
+main = (
```
Comment (willmcgugan) [inline naughty/subdomains.py]: Maybe make this a set literal?

Comment (willmcgugan) [inline naughty/subdomains.py]: Why call it 'main'? Are there going to be other lists of subdomains?

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+version = '0.0.1'
+
+
+setup(
+    author='WildFoundry Ltd',
+    author_email='info@wildfoundry.com',
+    classifiers=[
+        'Development Status :: 2 - Pre-Alpha',
+        'Intended Audience :: Developers',
+        'License :: OSI Approved :: GNU Lesser General Public License v3 (LGPLv3)',
```
Comment (willmcgugan) [inline setup.py:13]: Should we not use a less restrictive license as subdomains.py suggests?

Comment (meshy) [inline setup.py:13]: @willmcgugan: @barfle decided to use this license.

Comment (meshy) [inline setup.py:13]: We will be including content from other libraries with differing licenses. See #1.

File: naughty/subdomains.py (hunk 1 above)
Comment (meshy) [inline naughty/subdomains.py]: There will also be the fortune 500 list, and the "adult" list from the repo mentioned.

Comment (meshy) [inline naughty/subdomains.py]: I'm open to suggestions for names other than `main`, however.

Comment (meshy) [inline naughty/subdomains.py]: Sorry -- fortune 100.

**Comment by meshy** [comment]
@willmcgugan It's now updated to be a `set`, and I have added a test suite to boot.

File: tests/test_subdomains.py [hunk 3]
```diff
@@ -0,0 +1,8 @@
+import pytest
+
+from naughty import subdomains
+
+
+@pytest.mark.parametrize('words', [subdomains.main])
+def test_is_a_set(words):
+    assert isinstance(words, set)
```
Comment (meshy) [inline tests/test_subdomains.py:8]: ...If you can call this a test-suite...

**Comment by meshy** [comment]
@wildfoundry/web-developers bump

**Comment by willmcgugan** [comment]
:+1:

**Comment by meshy** [comment]
Ta

**Pull request merged**